### PR TITLE
[tbc-ge] Convert transaction time to local timezone

### PR DESCRIPTION
**Pull request by dzhiriki**

### CODE REVIEW

File: src/plugins/tbc-ge/converters.ts [hunk 1]
```diff
@@ -300,3 +300,17 @@ function parsePOSMonth (month: string): number {
       assert(false, 'cant parse pos month', month)
   }
 }
+
+// TBC API returns transaction date as midnight in UTC+4
+// But ZM uses the user's local timezone to set a date for operations
+// This causes a problem if the user is in a different timezone
+// That is also not a 100% correct solution,
+// but it should decrease the amount of operations with incorrect date
+export function convertToLocalTimezone (data: number): number {
```
Comment (Hermesiss) [inline src/plugins/tbc-ge/converters.ts:309]: Please see the method `parsePOSDateString` and its usages in the project, it should (in theory) already correct the offset

Comment (dzhiriki) [reply src/plugins/tbc-ge/converters.ts:309]: @Hermesiss Not exactly. It actually does quite the opposite.
The time in the POS string is always 12AM (midnight), and you use the set timezone as UTC+4.
But ZM uses the user's phone in the local timezone. So, for example, I did a bunch of transactions today (March 15th). All these transactions will have time as midnight of March 15th in UTC+4.
But if I am in UTC+2, then all these dates will be counted as March 14th, 22:00 in my local timezone, and ZM gonna save all today's transactions in the previous day.

TL;DR: if you are in a timezone to the west to UTC+4 you will have all transactions in the previous day.

Comment (dzhiriki) [reply src/plugins/tbc-ge/converters.ts:309]: @skvav Could you elaborate on why do you think that solution is better?
For me, it is kinda the same. The only difference is that we shift to a fixed amount, instead of the user's timezone difference

Comment (lucomsky) [reply src/plugins/tbc-ge/converters.ts:309]: @skvav  @dzhiriki 
I have the same issue in Credo Bank Georgia. All transactions time saved in local Georgian time zone UTC+4:00. So when I create transaction outside Georgia even Credo Bank application shows transaction as it was made MY_TIME_ZONE - UTC+4:00 timezone earlier.

Looks like no timezone part in timestamps from bank responses.
